docs/outlook/authenticate-a-user-with-an-sso-token.md

```diff
@@ -27,7 +26,7 @@ To use SSO, your Outlook add-in will need to have a server-side web API that is
 
 ### Provide consent when sideloading an add-in
 
-When an add-in that uses SSO is acquired from AppSource, it must have a backup authentication method for providing consent if it contains Microsoft Graph scopes. When you are developing an add-in, you will have to provide consent in advance. For more information, see [Grant administrator consent to the add-in](../develop/grant-admin-consent-to-an-add-in.md)
+When an add-in that uses SSO is acquired from AppSource, it must have a backup authentication method for providing consent if it contains Microsoft Graph scopes. When you are developing an add-in, you will have to provide consent in advance. For more information, see [Grant administrator consent to the add-in](../develop/grant-admin-consent-to-an-add-in.md).
```

Comment:
I think we should call out that the backup authentication method must exist no matter what (Graph scopes or not) as they add-in may be running on a client that doesn't support SSO or other reasons that cause the SSO call to fail.

Reply:
Actually, the mistake here is in even mentioning the back up system. The section title and subject matter are about providing admin consent for sideloading/developing. That's a different issue. So I'll just delete the entire first sentence.